src/app/containers/ComscoreAnalytics/utilities/index.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
```

Comment:
I'm not 100% sure what this file is from the filename!

Reply:
Yep I agree. I've re named it to testHelper and removed the utility folder as I see this being done in a few of our other containers.